components/organisms/SubdomainsTabBody.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, Modal, Skeleton } from "@ensdomains/thorin";
+import { Table } from "../02-molecules";
+import { DomainData } from "@/lib/domain-page";
+import { useState } from "react";
+import { CreateSubdomainModalContent } from "./CreateSubdomainModalContent";
+import { useRouter } from "next/router";
+
+export interface SubdomainsTabProps {
+  domainData: DomainData | null;
```

Comment:
what is the scenario where this is null?

Reply:
@pikonha when the domain is not registered